code/iaas/agent-instance/src/main/resources/META-INF/cattle/system-services/agent-instance-defaults.properties

```diff
@@ -9,7 +9,7 @@ agent.instance.start.items.increment.extra=
 agent.instance.services.base.items=${agent.instance.start.items.apply},${agent.instance.start.items.increment},${agent.instance.services.base.items.extra}
 agent.instance.services.base.items.extra=
 
-agent.instance.services.processes=nic.activate,instancelink.update,instancelink.activate,hostipaddressmap.activate,${agent.instance.services.processes.extra}
+agent.instance.services.processes=nic.activate,instancelink.update,instancelink.activate,hostipaddressmap.activate${agent.instance.services.processes.extra}
```

Comment:
Was there a problem with have the value end with a ","?  This changes means that if somebody used the extra field they would need to start with a ",".  In the end if the extra field is causing problem we can just removed the property.  It was a useless hook added for extension that is no longer needed.